ext/ddtrace_profiling_native_extension/stack_recorder.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <datadog/profiling.h>
+#include <ruby.h>
```

Comment:
Any LSP will complain about `VALUE` being undefined otherwise. It indeed isn't needed (at least at the moment) during compilation because all codepaths that include `stack_recorder.h` include `ruby.h` first. That being said, because of the `ifndef/define`, this extra include should have no impact on the compiled result.